docs/gen_edot_col_components/requirements.txt

```diff
@@ -1,2 +1,2 @@
 Jinja2==3.1.6
-PyYAML==6.0
```

Comment:
[for reviewer] I had to revert to an older version to avoid install issues, got this from https://github.com/yaml/pyyaml/issues/724.

Reply:
I also had this issue while trying to run the make command.

Reply:
Does 6.0.2 work?

Reply:
Yes, I hadn't tried and missed this option. Version is now 6.0.2 in 09bc9df